examples/with-docker/Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM node:14-alpine AS deps
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
 RUN apk add --no-cache libc6-compat
 WORKDIR /app
-COPY package.json ./
-RUN yarn install
+COPY package.json yarn.lock ./
```

Comment:
Hey @leerob , thank you for merging this PR.

If this line is added back without a yarn.lock file this is what will happen on docker build:

![Screen Shot 2021-06-08 at 12 56 53 pm](https://user-images.githubusercontent.com/170554/121116068-50e8a300-c859-11eb-9780-b2d4dcf70c93.png)

If it is expected to a yarn before running docker I did not find it in the readme. In addition, a CI system say Travis might just have docker and no node. I am not sure if this is what is intended. Happy to hear your thoughts!

Reply:
You can edit your code after cloning, but specifically for this example, we don't want the lockfile 👍

Reply:
Yes, so in this case the lock file shouldn't be referenced in the dockerfile too then?